kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -9,22 +9,25 @@ ssui_root="$repos_root/manageiq-ui-self_service"
 
 rm -rf $app_root
 
-git clone <%= @appliance_checkout.remote %> $appliance_root
+git clone <%= "--separate-git-dir /tmp/appliance.git" if @build_type == "release" %> \
+        <%= @appliance_checkout.remote %> $appliance_root
```

Comment:
I'm good with this as long as a darga build will be able to git pull darga updates, master git pull will get master updates, etc. 👍